docs/assemblies/proc_updating-the-data-plane-ovn.adoc

```diff
@@ -0,0 +1,68 @@
+[id="proc_updating-the-data-plane-ovn-{context}"]
+= Updating OVN on the data plane
+
+[role="_abstract"]
+
+You update OVN content (containers) on the data plane once OVN on the control
+plane has been updated.
+
+.Procedure
+
+. Validate that OVN has been updated on the control plane.
++
+----
+$ oc wait openstackversion <openstack_ctlplane_name> --for=condition=OpenStackVersionMinorUpdateOVNControlplane
```

Comment:
According tp my testing of https://github.com/openstack-k8s-operators/openstack-operator/pull/792/files this should be fixed to  `--for=condition=MinorUpdateOVNControlplane condition`

Reply:
indeed, thanks for spotting. fixed in https://github.com/openstack-k8s-operators/dataplane-operator/pull/929